src/content/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure.mdx

```diff
@@ -692,9 +692,9 @@ A basic user's abilities aren't directly defined by a specific role. A basic use
 
 ## Roles and capabilities [#roles]
 
-For users on the [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/transition-guide-new-pricing-user-changes#user-models), "roles" can be defined as "a set of capabilities." A capability is defined as the ability to do a specific New Relic task, like 'Delete alert conditions' ([learn more about capabilities](#capabilities)). 
+For users on the [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/transition-guide-new-pricing-user-changes#user-models), a "role" can be defined as "a set of capabilities." A capability is defined as the ability to do a specific New Relic task, like 'Delete alert conditions' ([learn more about capabilities](#capabilities)). 
```

Comment:
...like 'Delete alert conditions'. ([Learn more about capabilities](#capabilities)). - splitting in two sentences.